### DISCUSSION TITLE
Funnel

### DISCUSSION THREAD

**Discussion by Canic** [plausible/analytics#3319] [Self-Hosted Support]
Hello,

i have installed the latest docker image on my server and its working well.
I found on the docs site the feature "funnel", however its not visible on the docker image.
On the latest dev image "analytics-private" i can see the feature "Custom Properties" but not funnel.

Any ideas or do i am missing something?

**Comment by ruslandoga** [comment]
👋 @Canic 

The currently latest release `plausible/analytics:v2.0.0` doesn't yet include the funnel feature.

**Reply by Canic** [reply]
Do you already know when the feature is available, or is there any dev docker image where it's already implemented?

**Comment by StacySiz** [comment]
Any updates on Funnels for self hosted?

**Reply by ruslandoga** [reply]
👋 @StacySiz 

The currently latest release [v2.1.5](https://github.com/plausible/analytics/releases/tag/v2.1.5) doesn't yet include the funnel feature.

**Reply by madanyang** [reply]
So when will it be included ?

**Comment by aerosol** [comment]
Hi! Funnels are exclusive to Enterprise Edition. Currently it's not possible to purchase an EE license while self-hosting. To access premium features you need to use https://plausible.io/

**Reply by styxlab** [reply]
I am fine with hiding it behind a config option, but would be great if it was supported in CE as well.
